### PR TITLE
Simple menus were not enabled on windows by default

### LINKED ISSUE #1783: Windows detector is not actually used for selecting simple menus
Simple menus are only enabled when env `KONTENA_SIMPLE_TERM` is set. Not by default on windows as was intended.

```ruby
 35   def self.on_windows?
 36     ENV['OS'] == 'Windows_NT' && RUBY_PLATFORM !~ /cygwin/
 37   end
 38
 43   def self.simple_terminal?
 44     ENV['KONTENA_SIMPLE_TERM'] || !$stdout.tty?
 45   end
 46
 51   def self.prompt
 52     return @prompt if @prompt
 53     if simple_terminal?
 ```

Should be

```ruby
 44     on_windows? || ENV['KONTENA_SIMPLE_TERM'] || !$stdout.tty?
```

### PR DESCRIPTION
**Pull request by kke**
Fixes #1783

The "simple menus" were not used on Windows by default as  was intended. Only when you set env KONTENA_SIMPLE_TERM.

This PR makes windows choose the simple terminal always.

### CODE REVIEW

File: cli/lib/kontena_cli.rb [hunk 1]
```diff
@@ -41,7 +41,7 @@ def self.browserless?
   end
 
   def self.simple_terminal?
-    ENV['KONTENA_SIMPLE_TERM'] || !$stdout.tty?
+    on_windows? || ENV['KONTENA_SIMPLE_TERM'] || !$stdout.tty?
```
Comment (jakolehm) [inline cli/lib/kontena_cli.rb]: Test would be nice

File: cli/spec/kontena/kontena_cli_spec.rb [hunk 2]
```diff
@@ -2,27 +2,35 @@
 require 'kontena_cli'
 
 describe Kontena do
-  let(:subject) { described_class }
+  context 'prompt' do
+    it 'uses light prompt on windows' do
+      expect(ENV).to receive(:[]).with('OS').and_return('Windows_NT')
```
Comment (jnummelin) [inline cli/spec/kontena/kontena_cli_spec.rb]: `ENV` will get also other reads than `OS`, maybe use `allow` instead

Comment (kke) [inline cli/spec/kontena/kontena_cli_spec.rb]: using allow + expect.. funny it worked on ruby 2.3+ without the allow

**Comment by kke** [comment]
It still doesn't work.

**Pull request merged**